Controllers/freshvibesController.php

```
@@ -1509,4 +1528,52 @@
 			exit;
 		}
 	}
+
+	private function deduplicateLayout(array &$layout): void {
+		$globalSeenFeeds = [];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$globalSeenFeeds'. (Unused Code Rules)

(UnusedLocalVariable)